client/src/components/mobile/ResourceDetailMobile.js

```diff
@@ -0,0 +1,471 @@
+                Math.max(nowHour, endHour) === endHour;
+
+              const inMin =
+                Math.min(nowMin, startMin) === startMin &&
+                Math.max(nowMin, endMin) === endMin;
+
+              if (inHour) {
+                if (nowHour === startHour || nowHour === endHour) {
+                  withinHours =
+                    (nowHour === startHour &&
+                      Math.min(nowMin, startMin) === startMin) ||
+                    (nowHour === endHour &&
+                      Math.max(nowMin, endMin) === endMin);
+                } else {
+                  withinHours = true;
+                }
+              }
+            }
+          }
+
+          return { day, withinHours };
+        }),
+    );
+  }, [hours]);
+
+  if (!resourceExists) {
+    return <Redirect to="/resources/unknown" />;
+  }
+
+  const Map = ReactMapboxGl({
+    accessToken:
+      'pk.eyJ1IjoiYW5vb2psYWwiLCJhIjoiY2syemtiYjZoMGp1' +
+      'eDNscXQ3azJzajl0bCJ9.FDSFjP1IfSisbm4uvd70vg',
+    interactive: true,
+  });
+
+  /* SETUP END */
+
+  return (
+    <div className="mb-rd-container">
+      <Carousel className="mb-rd-carousel">
+        {image ? (
+          <img className="mb-rd-carousel-img" src={image} alt="" />
+        ) : (
+          <div />
+        )}
+      </Carousel>
+      <div className="mb-rd-block-container">
+        <div className="mb-rd-block-1">
+          <ResourcesBreadcrumb
+            categorySelected={category}
+            subcategorySelected={subcategory}
+            resourceSelected={name}
+          />
+          <Row className="mb-rd-header-bar" type="flex">
+            <Col>
+              <h2 className="mb-rd-header-text">{name}</h2>
+            </Col>
+            <Col>
+              <SaveButton
+                className="mb-rd-save-btn"
+                type="heart"
+                fontSize="2em"
+                authed={authed}
+                isSaved={isSaved}
+                deleteResourceHandler={deleteResourceHandler}
+                saveResourceHandler={saveResourceHandler}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Rate className="mb-rd-rate" disabled value={recommendation} />
+          </Row>
+          <Row className="mb-rd-description-container">
+            <Col className="mb-rd-description" span={20}>
+              {description}
+            </Col>
+          </Row>
+        </div>
+        <div className="mb-rd-block-2">
+          <Row className="mb-rd-block-title">Basic Information</Row>
+          <InfoBlock
+            title="Contact Information"
+            icon={
+              <Icon
+                className="mb-rd-phone-icon mb-rd-icon"
+                type="phone"
+                theme="filled"
+              />
+            }
+            content={[
+              phone && phone[0]
+                ? phone[0].phoneNumber
+                : 'No phone number provided.',
```

Comment:
Since phone is a list of phone numbers and types, this could end up not displaying some information.  For example if there was an "Office" phone number and a "Helpline" phone number entered.  Can we make it so it displays all?